src/structures/GuildChannel.js

```diff
@@ -253,6 +268,7 @@ class GuildChannel extends Channel {
         user_limit: data.userLimit != null ? data.userLimit : this.userLimit, // eslint-disable-line eqeqeq
         parent_id: data.parentID,
         lock_permissions: data.lockPermissions,
+        permission_overwrites: data.permissionOverwrites,
```

Comment:
Not documented for ChannelData

Reply:
Nope not yet. But if you open the inspect ui in the client and head over to the network monitor. That's how the client syncs permissions when you hit the sync button.

Reply:
Actually, I misunderstood your comment. No it isn't documented on the official api docs yet. But I just documented it in the edit method docs here at d.js.